docs/source/installation.rst

```diff
@@ -25,6 +26,10 @@ For a standard installation, execute:
 
     $ conda install -c conda-forge signac signac-flow
 
+.. note::
+
+    The synced_collections package is not yet available via conda and must be installed either from source or via pip.
```

Comment:
Let’s wait until packages are available to publish this. Then we can remove the note.